fs/fs_zenfs.h

```diff
@@ -353,6 +356,8 @@ class ZenFS : public FileSystemWrapper {
 #endif  // !defined(ROCKSDB_LITE) && defined(OS_LINUX)
 
 Status NewZenFS(FileSystem** fs, const std::string& bdevname);
+Status NewZenFS(FileSystem** fs, const std::string& bdevname,
+                std::shared_ptr<ZenFSMetrics> metrics);
```

Comment:
I think we should use just one constructor and then a default argument to `NoZenFSMetrics`, just as for `ZonedBlockDevice`

Reply:
Done.